gf2.cpp

```diff
@@ -842,7 +850,11 @@ void DebuggerGetBreakpoints() {
 
 			for (int i = 0; i < breakpoints.Length(); i++) {
 				if (strcmp(breakpoints[i].fileFull, breakpoint.fileFull) == 0 &&
+						strcmp(breakpoints[i].condition, breakpoint.condition) == 0 &&
 						breakpoints[i].line == breakpoint.line) {
+					char buffer[1024];
+					StringFormat(buffer, 1024, "delete %d", breakpoint.number);
+					DebuggerSend(buffer, true, false);
```

Comment:
here we could simply call `CommandDeleteBreakpoint(breakpoint.number);`, but for that we need to either forward declare the `CommandDeleteBreakpoint` or to put all function signatures in a header file (which will come in handy in the futur).